### ISSUE TITLE
s3 system test ./jenkins-build.sh failed 

### ISSUE THREAD

**Issue by yanqingfu** [Seagate/cortx-s3server#759]
Running auth_spec_negative_and_fi.py...
Traceback (most recent call last):
  File "auth_spec_negative_and_fi.py", line 27, in <module>
    from configobj import ConfigObj
ModuleNotFoundError: No module named 'configobj'

***************
*** ABORTED ***
***************

Error encountered. Exiting ST prematurely...

***************
*** ABORTED ***
***************

Error encountered. Exiting test runs prematurely...
echo 'S3 Tests failed.'
S3 Tests failed.

see detailed log 
[s3systemtestfailure.txt](https://github.com/Seagate/cortx-s3server/files/6134958/s3systemtestfailure.txt)

**Comment by sachitanands** [comment]
Hi @yanqingfu thanks for reporting this issue.. last night I ran the jenkins-build with latest changes and all tests were succeeded. 
hence could you please run below steps,
1) ./jenkins-build.sh --cleanup_only  (this will ensure cleanup of any inconsistent state )
2) ./jenkins-build.sh

**Comment by yanqingfu** [comment]
@sachitanands still face the same issue

Test case [No password value] - Execution time - 550 ms
Command has failed as expected.
Response has [Usage: java -jar AuthPassEncryptCLI-1.0-0.jar [options]].
Wed Mar 17 05:03:03 UTC 2021: Running auth_spec_negative_and_fi.py...
Traceback (most recent call last):
  File "auth_spec_negative_and_fi.py", line 27, in <module>
    from configobj import ConfigObj
ModuleNotFoundError: No module named 'configobj'

***************
*** ABORTED ***
***************

Error encountered. Exiting ST prematurely...

**Comment by sachitanands** [comment]
Hi @yanqingfu could you please check below steps,
1) what is the OS version you have and which template you are using from SSC portal for generating this vm
2) if its Centos 7.8 OS could you please try below command once,
  1)  sh cortx-s3server/scripts/env/dev/init.sh -a
  2)  sh cortx-s3server/jenkins-build.sh
  
If its still failing , is it possible to share the machine credentials with me?

**Comment by yanqingfu** [comment]
@sachitanands still fails
+ ./rebuildall.sh --no-motr-rpm --use-build-cache
cortx-s3server/jenkins-build.sh: line 362: ./rebuildall.sh: No such file or directory

**Comment by sachitanands** [comment]
Thanks @yanqingfu for sharing setup with me.
I tried running this ST on given setup and found the root cause of this ..
**Root cause**  : **configobj** package installation is missing in case of opensource ansible script, hence ST are breaking if someone use opensource setup. (i.e. running ansible script with "-a" option)
**Fix** :  **pip3 install configobj**
I have tested the patch locally on this setup and tests were successful.
hence will update the code to fix this issue and will raise a patch for the same.

FYI @nileshgovande 
![image](https://user-images.githubusercontent.com/65209830/111646363-9633a080-8827-11eb-88bd-8dccf1228ed7.png)
